build_scripts/wheels/macos.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eux
+
+brew update
+brew upgrade pyenv
+eval "$(pyenv init -)"
+export PYENV_ROOT="$HOME/.pyenv"
+export PATH="$PYENV_ROOT/bin:$PATH"
+pyenv --version
+pyenv install -s "$PYENV_VERSION"
+pyenv local "$PYENV_VERSION"
+python -V
+brew install -v bison || brew upgrade bison
+brew install -v cmake || brew upgrade cmake
+brew install -v ninja || brew upgrade ninja
+python -m pip install -U wheel setuptools
+export CMAKE_PREFIX_PATH="$(brew --prefix bison)"
+python setup.py bdist_wheel -d macos-wheelhouse
```

Comment:
I recommend replacing this with a PEP 517 build front-end.

Both `pip` and `pep517` will work (I tend to use `pep517.build` because it's at least designed for this, though it was somewhat grudgingly added).

`pip` version is:

```suggestion
export CMAKE_PREFIX_PATH="$(brew --prefix bison)"
python -m pip wheel . --no-deps -w macos-wheelhouse
```

The `pep517` version is:

```suggestion
python -m pip install -U pep517
export CMAKE_PREFIX_PATH="$(brew --prefix bison)"
python -m pep517.build -b -o macos-wheelhouse
```

Same comment applies in `manylinux.sh`.

Reply:
I had actually been trying to use `pip wheel` at first, but it was hanging for a very long time, I think because I had forgotten the `--no-deps` flag.  Thanks for catching that, will make the change.